plugin/pkg/scheduler/algorithm/scheduler_interface.go

```diff
@@ -18,10 +18,25 @@ package algorithm
 
 import (
 	"k8s.io/kubernetes/pkg/api"
+	schedulerapi "k8s.io/kubernetes/plugin/pkg/scheduler/api"
 )
 
+// SchedulerExtender is an abstract interface for external processes to influence scheduling
+// decisions made by Kubernetes.
+type SchedulerExtender interface {
+	// Filter based on extender implemented predicate functions. The filtered list is
+	// expected to be a subset of the supplied list.
+	Filter(pod *api.Pod, nodes *api.NodeList) (*api.NodeList, error)
+
+	// Prioritize based on extender implemented priority functions. Weight*priority is added
```

Comment:
s/extender implemented/extender-implemented/

I'm not sure the sentence "Weight*priority is added up for each such priority function" makes sense where it is, and should probably be deleted.